helm-charts/strimzi-kafka-operator/templates/07-crd-kafka-connect_s2i.yaml

```diff
@@ -0,0 +1,370 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kafkaconnects2is.kafka.strimzi.io
+  labels:
```

Comment:
This is tricky from the build perspective because the CRD resources are themselves generated from the Java model in `api`. We have to avoid having screeds of YAML which needs to be hand-maintained. We could perhaps generate the `examples` resources from the helm charts, but to do that for the CRD resources we'd need to be generating the resources with these extra labels. Which is doable. I assume apart from these Helm-specific labels the rest of the CRD templates are just a verbatim copy of what's currently in `examples`?

Reply:
@tombentley Yes I just copied the latest from `./examples/install/cluster-operator/`.  The main differences are the additions to `metadata` and Deployment in `08-deployment.yaml`.

Can you point me to the process in the build that generates the CRD?  I can take a crack at making these steps of the build process work together.